doc/source/examples/02-subset_assignment.py

```diff
@@ -0,0 +1,46 @@
+"""
+.. _ref_subset_assignment:
+
+Subset assignment
+=================
+
+This example demonstrates how to assign a subset 
+to a training data.
+
+"""
+
+###############################################################################
+# Import necessary libraries
+# --------------------------
+
+import ansys.simai.core as asc
+
+simai = asc.from_config()
+
+###############################################################################
+# Select a training data
+# ----------------------
+# Example of a training_data_id associated with a project_id
+
+training_data_id = "k4z77qzq"
+project_id = "k9756vw0"
+
+###############################################################################
+# Get subset assignment
+# ---------------------
+# Get the current subset assigned for this training_data_id.
+
+simai.training_data.get(id=training_data_id).get_subset(project=project_id)
```

Comment:
This does nothing for the user as this will return a subset but we don't assign it to a variable or print it.
Maybe
```suggestion
# Get and print the current subset assigned for this training_data_id.

current_subset = simai.training_data.get(id=training_data_id).get_subset(project=project_id)
print(current_subset)
```